eng/pipelines/templates/jobs/archetype-sdk-client.yml

```diff
@@ -88,16 +95,16 @@ jobs:
         inputs:
           packageType: sdk
           version: "$(DotNetCoreSDKVersion)"
-      - script: "dotnet test eng/service.proj --filter TestCategory!=Live --logger trx /p:ServiceDirectory=${{parameters.ServiceDirectory}}"
-        displayName: "Build & Test"
+      - script: dotnet test eng/service.proj --filter TestCategory!=Live --framework $(TFM) --logger "trx;LogFileName=$(TFM).trx" /p:ServiceDirectory=${{ parameters.ServiceDirectory }} /p:IncludeSrc=false
+        displayName: "Build & Test ($(TFM))"
         env:
           DOTNET_SKIP_FIRST_TIME_EXPERIENCE: 1
           DOTNET_CLI_TELEMETRY_OPTOUT: 1
           DOTNET_MULTILEVEL_LOOKUP: 0
       - task: PublishTestResults@2
-        condition: succeededOrFailed()
```

Comment:
We should keep this condition so that test results still get uploaded if the test command fails.